library/ui/src/main/java/com/google/android/exoplayer2/ui/SpannedToHtmlConverter.java

```diff
@@ -197,6 +198,14 @@ private static String getOpeningTag(Object span, float displayDensity) {
       }
     } else if (span instanceof UnderlineSpan) {
       return "<u>";
+    } else if (span instanceof TextEmphasisSpan) {
+      TextEmphasisSpan textEmphasisSpan = (TextEmphasisSpan) span;
+      String style = getTextEmphasisStyle(textEmphasisSpan.mark);
+      String position = getTextEmphasisPosition(textEmphasisSpan.position);
+      return Util
+          .formatInvariant("<span style='-webkit-text-emphasis-style: %s; text-emphasis-style: %s; "
+                  + "-webkit-text-emphasis-position: %s; text-emphasis-position: %s;'>",
```

Comment:
(I'm not a web developer, apologies if this is a silly question)

If you know that this is always going to be used in a WebView, do you need to use the `-webkit` prefixed styles as well?

This table suggests `text-emphasis-style` (I assume without the webkit prefix) is supported from 4.4 (i.e. KitKat):
https://developer.mozilla.org/en-US/docs/Web/CSS/text-emphasis-style#browser_compatibility

Is this to make things also work on Jelly Bean (ExoPlayer's min supported version)?

I'm mostly asking in case we should add the same prefix for other CSS in this file...

Reply:
The way they wrote it in the documentation is a little confusing but in the legend section, you'll see that it says
'-' or 'x-' - requires a vendor prefix or different name to use. 

If you try this code on the chrome browser, you'll see that the text emphasis is not rendered correctly on Chrome without the webkit prefix, but renders correctly on Firefox or Safari.

```
<body>
  <span style='text-emphasis: filled double-circle blue;'>Text emphasis</span>
</body>
```

Reply:
Got it, thanks!